src/__tests__/proposalVersions.int.test.ts

```diff
@@ -140,22 +139,22 @@ describe('/proposalVersions', () => {
 				})
 				.expect(201);
 			const after = await loadTableMetrics('proposal_field_values');
-			logger.debug('after: %o', after);
+
 			expect(before.count).toEqual(0);
 			expect(result.body).toMatchObject({
 				id: 1,
 				proposalId: 1,
 				fieldValues: [
 					{
-						id: 1,
+						id: expect.any(Number) as number,
 						applicationFormFieldId: 1,
 						position: 1,
 						value: 'Gronald',
 						isValid: true,
 						createdAt: expectTimestamp,
 					},
 					{
-						id: 2,
+						id: expect.any(Number) as number,
```

Comment:
This makes the test pass, but we do know in advance what ID to expect, right? Why not make the inserts sequential? Then again, these are expected to be auto-generated, so we do not necessarily know in advance. OK, fine! : )

Reply:
Unfortunately we don't know what ID to expect in advance right now because *SOMEONE* thought it'd be smart to insert in parallel (it was me, sorry).

I agree that's not a useful optimization, but I didn't want to change behavior in the name of fixing a test and opted to fix the test for now.

I expect to fix insertion soon (at which point I'll make the test better)

Reply:
*SOMEONE* is the worst, aren’t they?